[2018-07-16 14:32:45.568] [ERROR] dateFileLog - [Error: Not found this user.]
Error: Not found this user.
    at callback_getUser (D:\NB\yujiang.Foil.Node.BizServer\biz\account\checkLogin.js:106:17)
    at D:\NB\yujiang.Foil.Node\src\yjError.js:134:4
    at Object.exports.safeSuccess (D:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.safeCallback (D:\NB\yujiang.Foil.Node\src\yjError.js:133:11)
    at Object.exports.handleResult (D:\NB\yujiang.Foil.Node\src\yjError.js:148:11)
    at Query.<anonymous> (D:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:202:38)
    at bound (domain.js:287:14)
    at Query.runBound [as _callback] (domain.js:300:12)
    at Query.Sequence.end (D:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (D:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
[2018-07-16 14:32:45.586] [ERROR] dateFileLog - [Error: 没有找到此用户。]
Error: 没有找到此用户。
    at Request.<anonymous> (D:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
    at Request.mixin._fireSuccess (D:\NB\node_modules\restler\lib\restler.js:220:12)
    at D:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.parsers.auto (D:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request.mixin._encode (D:\NB\node_modules\restler\lib\restler.js:195:29)
    at D:\NB\node_modules\restler\lib\restler.js:154:16
    at Request.mixin._decode (D:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (D:\NB\node_modules\restler\lib\restler.js:147:14)
